Workbooks/CosmosDb/Overview/Overview.workbook

```diff
@@ -416,7 +378,11 @@
       "content": {
         "json": "### Cosmos DB account details: <span style=\"color:blue\"> _{ResourceId:name}_ </span> "
```

Comment:
same comment as the other PR, might want to try this on dark theme and see if there's a better way

Reply:
Removed HTML color code in markup